docs/usage.md

```diff
@@ -1,61 +1,94 @@
-# differentialabundance/differentialabundance: Usage
+# nf-core/differentialabundance: Usage
+
+## :warning: Please read this documentation on the nf-core website: [https://nf-co.re/differentialabundance/usage](https://nf-co.re/differentialabundance/usage)
 
 > _Documentation of pipeline parameters is generated automatically from the pipeline schema and can no longer be found in markdown files._
 
 ## Introduction
 
-<!-- TODO nf-core: Add documentation about anything specific to running your pipeline. For general topics, please point to (and add to) the main nf-core website. -->
+Differential analysis is a common task in a variety of use cases. In essence, all these use cases entail taking an input matrix containing features (e.g. genes) and observations (e.g. samples), and comparing groups of observations in all or a subset of the features. The feature/ observation language here reflects our hope that this workflow will extend in future to encompass a variety of applications where an assumption of gene vs sample may not be a valid one- though that is the application to which the first release will apply.
+
+With the above in mind, running this workflow requires:
 
-## Samplesheet input
+- a matrix of quantifications with observations by column and features by row
+- a description of the observations such as a sample sheet from RNA-seq analysis
+- a description of the features, for our initial RNA-seq application this is simply the GTF file from which gene annotations can be derived
+- a specification of how the matrix should be split, and how the resulting groups should be compared
 
```

Comment:
```suggestion
Currently, this limits the usage of differentialabundance to genomics applications where a GTF annotation file is available. Future developments of the pipeline will encompass changes that enable users to use the pipeline on more generic cases.
```

Reply:
Thanks! I can't apply the suggestion here, but [adding it to a separate PR against dev](https://github.com/nf-core/differentialabundance/pull/48)